### ISSUE TITLE
dub build failure

### ISSUE THREAD

**Issue by btbytes** [trishume/ddbus#1]
```
ddbus git:(master) dub build 
Package dunit can be upgraded from 1.0.10 to 1.0.11.
Use "dub upgrade" to perform those changes.
Target dunit 1.0.10 is up to date. Use --force to rebuild.
Building ddbus 1.0.1 configuration "application", build type debug.
Compiling using dmd...
source/ddbus/router.d(127): Error: template instance groupBy!((a) => a.iface) template 'groupBy' is not defined, did you mean group(alias pred = "a == b", Range)(Range r)?
FAIL .dub/build/application-debug-posix.osx-x86_64-dmd_2068-7887E279ED85A532A862ACB060F61534/ ddbus executable
Error executing command build:
dmd failed with exit code 1.
```

**Comment by btbytes** [comment]
I was able to verify that the same error occurs on CentOS / D2.068.2. (The above error message is from a Mac OS X 10.10.x and D2.068.

**Comment by trishume** [comment]
derp probably a D version issue.

**Comment by trishume** [comment]
@btbytes I think I fixed it in a branch if you are still interested in using the library. https://github.com/trishume/ddbus/compare/D2.069.2?expand=1#diff-d6a039ab592299e67f73e16e834bb59fR126 

Unfortunately the DBus service on my computer has stopped working since I last worked on ddbus for no apparent reason. It will no longer start, so I can't test that ddbus still works with the change I made. It should though. I guess I'll have to get around to switching to Linux away from OSX some day to avoid these issues.
